sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -9,6 +9,7 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableClientSdkAnalyzers>false</EnableClientSdkAnalyzers>
+    <EnableApiCompat>false</EnableApiCompat>
```

Comment:
@pakrym recommended this as a workaround for what he confirmed as a false positive, and this was necessary to pass CI. The false positive was to do with the changes in core we decided to revert, though, so we can turn it on with that revert, instead of waiting until everything else was approved and passing.